lib/active_model/serializer.rb

```diff
@@ -50,6 +50,10 @@ def self.type(type)
       self._type = type
     end
 
+    def self.links(&block)
+      self._links_block = block
+    end
```

Comment:
only if we design it this way.  simple solution is just pass in `links` to the adapter until we need more